.vscode/settings.json

```diff
@@ -0,0 +1,10 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "files.exclude": {
+    "**/.git": true,
+    "**/.svn": true,
+    "**/.hg": true,
+    "**/.DS_Store": true,
+    "**/build": true
+  }
+}
```

Comment:
Done, see 13f3164.

Do you want me to open another PR for these settings? They can help other people to get started more quickly when contributing to the project (allow debugging unit tests out of the box), and the maintenance required for these settings is minimal.